src/main/kotlin/eu/europa/ec/eudi/openid4vci/Config.kt

```diff
@@ -57,6 +60,20 @@ data class OpenId4VCIConfig(
     }
 }
 
+/**
+ * Wallet's policy in regard to using PAR, during a authorization code grant.
+ * - [IfSupported]: If authorization server advertises PAR endpoint it will be used. Otherwise, falls back
+ *   to usual authorization code flow
+ * - [Never]: Disables PAR. Wallet will use the usual authorization code flow
+ * - [Always]: Wallet always will place PAR request, regardless what if authorization server advertises the PAR
+ *   endpoint
+ */
+enum class ParUsage {
+    IfSupported,
+    Never,
+    Always,
```

Comment:
Perhaps `Always` could be renamed to `Required` instead?

Reply:
Done